### PR TITLE
created homecontroller and home html file

### PR DESCRIPTION
**Pull request by JBajracharya**

### CODE REVIEW

**Review by holliemaethomas — DISMISSED** [review]

**Comment by holliemaethomas** [comment]
k

**Comment by holliemaethomas** [comment]
Merge is fine

**Review by phongvdoan — CHANGES_REQUESTED** [review]
Need Login form

File: src/main/resources/templates/home.html [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <h1>GotYourBAC</h1>
+        <h2>Log in</h2>
```
Comment (phongvdoan) [inline src/main/resources/templates/home.html:14]: Login form needs to be created.

**Review by phongvdoan — CHANGES_REQUESTED** [review]
Lets get this completed

**Pull request closed without merging**